modules/services/mpris-proxy.nix

```
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.mpris-proxy;
+
+in {
+  meta.maintainers = [ maintainers.thibautmarty ];
+
+  options.services.mpris-proxy.enable = mkEnableOption
+    "a proxy forwarding bluetooth midi controls via mpris2 to control media players";
```

Technically, MIDI and MPRIS2 are abbreviations.

```suggestion
    "a proxy forwarding Bluetooth MIDI controls via MPRIS2 to control media players";
```